libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -354,6 +354,8 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
       const isSelected = isSameDay(this.selectedDate, cellDate);
       const cell = {
```

Comment:
Consider adding type here as well:
```suggestion
      const cell: CalendarCell = {
```